add initial modules animation

mixcas opened the pull request.

Commit:
add initial modules animation

varffs commented on src/js/main.js:
```diff
@@ -296,7 +297,47 @@ Site.Modules = {
+      setTimeout( function(callback) {
+        video.play();
+        setTimeout( function() {
+          video.pause();
```
I don't think this works. It just feels broken rather than like a preview. Especially as there is no clear sign that hover is the play state. I think this should play all the way thru once and then reset. That helps solve the mobile issue too.

mixcas replied:
When you say "I don't think this works" you mean the code or that the animation doesn't communicate what we are trying to say?
You are right, playing the whole video might look better.

varffs replied:
Sorry yes I dont think this is good UX. I think it should play all the way thru

varffs commented on src/js/main.js:
```diff
@@ -296,7 +297,47 @@ Site.Modules = {
+  playAnimation: function() {
+    var _this = this;
+
+    _this.played = true;
```
I think this is a bit of an issue too. Maybe fine when playing fully once. Otherwise weird is doesnt retrigger

mixcas replied:
What do you mean an issue? The animation should never retrigger.

varffs replied:
Why should it never retrigger? The intention is just for these to play the first time down on the first scroll and then not again?

mixcas replied:
yeah

varffs commented on src/js/main.js:
```diff
@@ -274,17 +274,18 @@ Site.Countdown = {
     _this.$videos = $('.module-video');
 
-    // TODO: Launch enter animation
+    _this.threshold = _this.$videos.offset().top - ($(window).height() / 3) * 2;
```
This doesn't handle a resized window. But is a solution to the splash video trigger I think.

mixcas replied:
True, this needs to respond to resize.

Commit:
add play thru all modules video

Commit:
rename to offset

mixcas commented:
I changed it so the modules videos play complete once. Also added a parameter we can use test with different offset between the videos. I think we should try it with the final videos.

Commit:
fix animation offset

Commit:
add conditions for empty module fields, close #21

Commit:
center CENTRAL MODULE text, close #23

Commit:
bind module hover after first play

Commit:
add comment to module video offset variable
